spec/views/content_items/index.html.erb_spec.rb

```diff
@@ -1,23 +1,27 @@
 require 'rails_helper'
 
 RSpec.describe 'content_items/index.html.erb', type: :view do
-  let(:content_items) { build_list(:content_item, 2) }
+  let(:organisation) { create(:organisation_with_content_items, content_items_count: 4) }
```

Comment:
If we increase the number of items when setting up a test, we need a reason for it. Adding two more items in each test would imply creating more records in the database in each test, hence, we are slowing our tests.